src/databricks/labs/ucx/progress/jobs.py

```diff
@@ -48,7 +50,36 @@ def _job_problems(self) -> dict[int, list[str]]:
             index[job_problem.job_id].append(failure)
         return index
 
+    @cached_property
+    def _direct_fs_accesses(self) -> dict[str, list[str]]:
```

Comment:
Why should we keep it as a cached property?

Reply:
To compute the property only once